src/modules/block/components/blockDetails/blockDetails.test.js

```diff
@@ -1,32 +1,19 @@
-import { mountWithRouter } from 'src/utils/testHelpers';
+import { mountWithRouter, mountWithQueryClient } from 'src/utils/testHelpers';
 import { truncateAddress } from '@wallet/utils/account';
-import blocks from '@tests/constants/blocks';
 import transactions from '@tests/constants/transactions';
+import { mockTransactions } from '@transaction/__fixtures__';
+import { useTransactions } from '@transaction/hooks/queries/useTransactions';
 import BlockDetails from './blockDetails';
 
+jest.mock('@transaction/hooks/queries/useTransactions');
```

Comment:
query mock is not allowed here modify the msw inside the test or change it on mock based on query params
```
 server.get("/api/**", () => {
      return res(
        ctx.json({
          data: mockTransactions.data[0],
        })
      );
    })
    ```